### PR TITLE
Tooltips and chart fixes

### PR DESCRIPTION
**Pull request by rihorn2**
Add formatted tooltips to all charts (except box plots, which are not customizable in plotly at this time). 
Set better defaults for charts.
Add 'count' option to data profile histogram view.
Fix bugs encountered while testing

### CODE REVIEW

File: visualization/dashboard/src/Localization/en.json [hunk 1]
```diff
@@ -234,7 +234,9 @@
         "falsePositive": "False positive",
         "falseNegative": "False negative",
         "dataset": "Dataset",
-        "predictedProbabilities": "Prediction probabilities"
+        "predictedProbabilities": "Prediction probabilities",
+        "none": "Count",
+        "_none.comment": "option to not have data on this axis, instead just counts number of points"
```
Comment (imatiach-msft) [inline visualization/dashboard/src/Localization/en.json:239]: just counts number -> just count number

File: visualization/dashboard/src/MLIDashboard/Cohort.ts [hunk 2]
```diff
@@ -7,9 +7,9 @@ export class Cohort {
     private static _cohortIndex: number = 0;
 
     public rowCount: number = 0;
+    public filteredData: Array<{[key: string]: number}>;
```
Comment (imatiach-msft) [inline visualization/dashboard/src/MLIDashboard/Cohort.ts:10]: very cool type, "Array<{[key: string]: number}>"!  that's an array of dictionaries in typescript?

File: visualization/dashboard/src/MLIDashboard/Controls/DatasetExplorerTab/DatasetExplorerTab.tsx [hunk 3]
```diff
@@ -505,33 +518,42 @@ export class DatasetExplorerTab extends React.PureComponent<IDatasetExplorerTabP
...
                     if (chartProps.xAxis.options.dither) {
-                        hovertemplate += "x: %{customdata.X}<br>";
+                        hovertemplate += xName + ": %{customdata.X}<br>";
                     } else {
-                        hovertemplate += "x: %{x}<br>";
+                        hovertemplate += xName + ": %{x}<br>";
                     }
                 }
                 if (chartProps.yAxis) {
                     if (chartProps.yAxis.options.dither) {
```
Comment (imatiach-msft) [inline visualization/dashboard/src/MLIDashboard/Controls/DatasetExplorerTab/DatasetExplorerTab.tsx:539]: the y axis and x axis case seem similar, consider refactoring the code out to a parameterized method, similarly the 
```
const xName = jointData.metaDict[chartProps.xAxis.property].abbridgedLabel;
const yName = jointData.metaDict[chartProps.yAxis.property].abbridgedLabel;
```

File: visualization/dashboard/src/MLIDashboard/Controls/DependencePlot/DependencePlot.styles.ts [hunk 4]
```diff
@@ -73,7 +73,7 @@ export const dependencePlotStyles: () => IProcessedStyleSet<IDependencePlotStyle
         },
         secondaryChartPlacolderBox: {
             height: "400px",
-            width: "100%"
+            flex: 1
```
Comment (imatiach-msft) [inline visualization/dashboard/src/MLIDashboard/Controls/DependencePlot/DependencePlot.styles.ts:76]: interesting!  haven't seen this before.  are there other flexible items?  Is this referring to: https://www.w3schools.com/cssref/css3_pr_flex.asp
how does this impact the chart as opposed to width=100%?

File: visualization/dashboard/src/MLIDashboard/Cohort.ts (hunk 2 above)
Comment (rihorn2) [reply visualization/dashboard/src/MLIDashboard/Cohort.ts:10]: yes, an array of dictionaries, where the keys are strings and the values are numbers

Comment (rihorn2) [reply visualization/dashboard/src/MLIDashboard/Cohort.ts:10]: typescript is great, welcome!

File: visualization/dashboard/src/MLIDashboard/Controls/DependencePlot/DependencePlot.styles.ts (hunk 4 above)
Comment (rihorn2) [reply visualization/dashboard/src/MLIDashboard/Controls/DependencePlot/DependencePlot.styles.ts:76]: it is referring to flex! here is a helpful guide https://css-tricks.com/snippets/css/a-guide-to-flexbox/
@imatiach-msft

**Comment by rihorn2** [comment]
rebased and this branch died.

**Pull request closed without merging**